tests/pkg_libcose/Makefile

```diff
@@ -0,0 +1,17 @@
+include ../Makefile.tests_common
+
+# HACL* only compiles on 32bit platforms
+BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno \
+                   jiminy-mega256rfr2 mega-xplained waspmote-pro \
+                   chronos msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
+
+TEST_ON_CI_WHITELIST += all
+
+CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(4*THREAD_STACKSIZE_DEFAULT\)
```

Comment:
Might the stacksize in the unittests be reduced then maybe?

Reply:
I'm afraid not, looking at [this] commit, the stacksize required this size because of the tweetnacl unittest. That commit is from before libcose was included in the unittests.

[this]: https://github.com/RIOT-OS/RIOT/commit/809c4b772450c2261462b31db3d6c99ecdde1eec

Reply:
Ok. Maybe we should move `tweetnacl` out of the unittests as well (but this is OOSOTPR (out-of-scop of this PR)